keyvalue/clients/src/main/java/tachyon/client/keyvalue/hadoop/KeyValueOutputCommitter.java

```diff
@@ -32,85 +38,62 @@
  * Extension of {@link FileOutputCommitter} where creating, completing, or deleting a
  * {@link KeyValueStores} in different phases of a job's or task's lifecycle is considered.
  * <p>
- * This committer is forced to be used in {@link KeyValueOutputFormat} (no matter what users have
- * set as the {@link org.apache.hadoop.mapred.OutputCommitter} in configration) to merge the
- * key-value stores created by each Reducer into one key-value store under the MapReduce output
- * directory.
+ * This committer must be used along with {@link KeyValueOutputFormat} to merge the key-value stores
+ * created by each Reducer into one key-value store under the MapReduce output directory.
  */
-class KeyValueOutputCommitter extends FileOutputCommitter {
+public class KeyValueOutputCommitter extends FileOutputCommitter {
```

Comment:
Since this is public API we might want to leave it non-final so that others can extend it. @yupeng9?